http-server/pom.xml

```diff
@@ -183,7 +183,12 @@
         <dependency>
             <groupId>com.facebook.airlift</groupId>
             <artifactId>http-client</artifactId>
-            <scope>test</scope>
+            <exclusions>
```

Comment:
I don't think it is a good idea to include the `http-client` into the `http-server` module. We are specifically trying to avoid that. Please create a separate module (e.g.: `http-utils`), add the `jetty-util` library as a dependency, and move the `ConcurrentScheduler` class there. Then add the `http-utils` class as a dependency for the `http-client` and `http-server` modules.